consensus/leader_election/vrf/vrf.go

```diff
@@ -0,0 +1,98 @@
+package vrf
+
+// This file is a light wrapper around https://pkg.go.dev/github.com/ProtonMail/go-ecvrf.
+// It is needed to to achieve more semantic variable naming in the use of HotPocket.
+
+import (
+	"bytes"
+	"crypto/sha512"
+	"fmt"
+	"io"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+
+	"github.com/ProtonMail/go-ecvrf/ecvrf"
```

Comment:
Is there documentation justifying the use of this dependency vs alternatives?

Reply:
Might want to take a look at this doc for reference on this: https://github.com/pokt-network/pocket/tree/main/docs/deps

Reply:
Done!